ui/raidboss/skins/dorgrin/Readme.md

```diff
@@ -0,0 +1,32 @@
+# Dorgrin's Raidboss Customisations
+Some customisations I mate to make raidboss a bit nicer on my eyes. Changes include colours, font face, bar and alert sizes.
+
+Warnings are larger and have a translucent background to make them more easily noticable. Timelines are taller, wider, and brighter.
+
+A custom font is used, which is optional. See below for installation instructions and source.
+
+## Screenshots
+### Alerts/Warnings
+These come in purple, orange and red.
+
+##### with default Font
+![Critical Warning](https://imgur.com/BOqm6cY)
+![Medium Warning](https://imgur.com/ZE8VI04)
+![Standard Warning](https://imgur.com/m3YAip4)
+
+##### with Peace Sans Font
+![Critical Warning](https://imgur.com/EbuHsMV)
+![Medium Warning](https://imgur.com/70qDlv3)
+![Standard Warning](https://imgur.com/khnZb3Z)
+
+### Timelines
+![Timelines featuring an active ability](https://imgur.com/XScpnXi)
+
+## Peace Sans Font Installation
+If you'd like to use the custom font, just follow these instructions. The stylesheet tries to use it; if it's not available, it'll fallback to default.
+1. Go to https://www.dafont.com/peace_sans.font
```

Comment:
Is it possible to include the font directly in the repository and use https://developer.mozilla.org/en-US/docs/Web/CSS/@font-face `url` to reference it? That way folks don't have to install anything and it will "just work".

Reply:
I shall give it a go

Reply:
haha ok, if it's tricky let me know and I'll take a look

Reply:
Nah should be doable, just so busy this past week :'(